### PR TITLE
Fix whitelist typo

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 lowercaseOutputName: true
 lowercaseOutputLabelNames: true
-whilelistObjectNames:
+whitelistObjectNames:
   - ^org.apache.camel:
```
Comment (rhuss) [inline java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml:5]: Need a list here ;-) (hint: quotes ;-)

Comment (jimmidyson) [inline java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml:5]: Sorry - copy/paste errors...

**Comment by rhuss** [comment]
@jimmidyson sorry, I have to revert the PRCbecause jmx_exporter doesnt allow patterns in the whiletlist: https://github.com/prometheus/jmx_exporter/blob/master/collector/src/main/java/io/prometheus/jmx/JmxCollector.java#L96-L100

;-(

Until this is fixed, its probably not possible to add a generic purpose prometheus filter here ....

**Comment by rhuss** [comment]
we can use an MBean pattern, though. just readded it.

**Pull request merged**